integration_tests/header.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
do we want this script to do something like a `go install`, to ensure one is not running the tests with an old `dealbot` binary?

Reply:
I was thinking of doing that in `.circleci/config.yml` - not sure if the test itself should be responsible for the building of the dependencies.

Reply:
personally I think CI should be as thin as possible; running the tests should be easy, akin to just `go test ./...`. Plus, a `go install` should be practically instantaneous if nothing changed :)

Reply:
Fair, we can leave that for a future "quality of life" improvement.